src/modules/vault/pages/balance/index.tsx

```diff
@@ -135,22 +136,54 @@ const VaultBalancePage = () => {
       </HStack>
 
       <Flex w="full" direction="column" flex={1}>
-        <Box mb={5} w="full">
-          <Text
-            color="grey.50"
-            fontWeight="bold"
-            fontSize={{ base: 'sm', sm: 'unset' }}
-          >
-            Balance
-          </Text>
-        </Box>
+        <Flex borderBottom="1px solid #333" w="full" mb={5}>
+          <Flex gap={2}>
+            <Box
+              px={5}
+              py={2}
+              fontSize="sm"
+              cursor="pointer"
+              borderTopLeftRadius="lg"
+              borderTopRightRadius="lg"
+              bg={activeTab === 'assets' ? 'white' : '#201F1D'}
+              color={activeTab === 'assets' ? 'black' : 'white'}
+              onClick={() => setActiveTab('assets')}
+            >
+              Tokens
+            </Box>
+            <Box
+              px={5}
+              py={2}
+              fontSize="sm"
+              cursor="pointer"
+              borderTopLeftRadius="lg"
+              borderTopRightRadius="lg"
+              bg={activeTab === 'nfts' ? 'white' : '#201F1D'}
+              color={activeTab === 'nfts' ? 'black' : 'white'}
+              onClick={() => setActiveTab('nfts')}
+            >
+              NFT
+            </Box>
+          </Flex>
+        </Flex>
 
         <CustomSkeleton
           isLoaded={!userInfos.isLoading && !assets.isLoading}
           flex={1}
         >
-          {hasAssets ? (
-            <AssetsBalanceList assets={assets.assets!} nfts={assets.nfts!} />
+          {activeTab === 'assets' ? (
```

Comment:
i suggest you create custom component to handle tabs or use chakra tabs https://v2.chakra-ui.com/docs/components/tabs/usage

```suggestion
interface TabPanelProps {
  children?: React.ReactNode;
  index: number | string;
  value: number | string;
}

export const TabPanel = ({
  children,
  value,
  index,
  ...other
}: TabPanelProps) => {
  return (
    <div
      role="tabpanel"
      hidden={value !== index}
      aria-labelledby={`tab-${index}`}
      {...other}
    >
      {value === index && <Box>{children}</Box>}
    </div>
  );
};

```